src/python/gudhi/representations/metrics.py

```diff
@@ -10,6 +10,10 @@
 import numpy as np
 from sklearn.base import BaseEstimator, TransformerMixin
 from sklearn.metrics import pairwise_distances
+from gudhi.wasserstein import wasserstein_distance as pot_wasserstein_distance
```

Comment:
This one could fail if POT is not installed, and currently we are trying to consider POT as optional. If it isn't too hard, it could be nice to handle that gracefully (otherwise, it isn't so bad).

Reply:
OK so I raised an exception if POT is not installed. Is this what you had in mind?